setup.cfg

```
@@ -49,7 +49,7 @@ include_package_data = True
 packages = find:
 python_requires = >=3.6
 install_requires =
-	jupyter_server~=1.0
+	jupyter_server>=1.0
```

Maybe we can use spaces here so it's correctly aligned with the other dependencies:

```suggestion
    jupyter_server>=1.0
```

---

Thanks.